spec/controllers/review_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../spec_helper'
+
+describe ReviewsController do
+  describe 'POST create;' do
+    it 'saves a review' do
+      initial_reviews_count = Review.all.count
+      video = Fabricate(:video)
+      review = Fabricate(:review)
+      post :create, {
+                      video_id: video.id,
+                      review: {
+                                body:review.body,
+                                title: review.title,
+                                rating: review.rating
+                              }
+                    }
+      expect(Review.all.count).to be > initial_reviews_count
```

Comment:
I see now, you are creating a `review` prior posting to `:create` this is not the right way. You are supposed to test whether the controller is supposed to create a new review. `Fabricate(:review)` on line 8 - is already creating one. So basically you are creating a copy of the same review.

My suggestion would be also this:

``` ruby
it "creates a review" do
  video = Fabricate(:video)
  expect {
    post :create, video_id: video.id, review: Fabricate.attributes_for(:review)
  }.to change { Review.count }.by 1
end
```

You are using rspecs `change` matcher to compare whether there as a new addition to `Review.count`.